screens/AuthScreen.styled.ts

```diff
@@ -2,46 +2,52 @@ import styled from 'styled-components/native';
 
 import Colors from '../theme/Colors';
 
-const StyledView = styled.View`
+const StyledContainer = styled.View`
   flex: 1;
-  align-items: center;
+  flex-direction: column;
   justify-content: center;
-  background-color: white;
+  align-items: center;
+  padding: 20px;
```

Comment:
I've got an idea: we can define some default values for padding and other metrics like: `padding-sm`, `padding-md`, `padding-lg` etc. in one module. What do you think?